package.json

```diff
@@ -30,11 +30,11 @@
     "react-visjs-timeline": "^1.5.0",
     "redux": "^4",
     "redux-thunk": "^2.2.0",
-    "semantic-ui-css": "^2.3.1",
+    "semantic-ui-css": "^2.4.0",
     "semantic-ui-react": "^0.82.3",
     "stable": "^0.1.8",
     "toposort": "^2",
-    "url-search-params-polyfill": "^4.0.1",
+    "url-search-params-polyfill": "^5.0.0",
```

Comment:
This is not actually a breaking change, just was accidentally published as one 🤷‍♀️